src/main/java/core/basesyntax/Return.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import dao.WareHouseDao;
+import dao.impl.WareHouseDaoImpl;
+
+public class Return extends OperationStrategy {
+    @Override
+    public void makeOperation() {
+        WareHouseDao wareHouseDao = new WareHouseDaoImpl();
```

Comment:
Don't create dao for each method call in each strategy handler. Consider making it a constructor injected dependency or smth alike